solara/autorouting.py

```diff
@@ -313,6 +316,11 @@ def get_page(module: ModuleType, required=True):
     return page
 
 
+def get_markdown_renderer(module: ModuleType):
+    markdown_renderer = getattr(module, "MarkdownRenderer", None)
+    return markdown_renderer
+
+
```

Comment:
```suggestion
```
Lets do this as minimal as possible, I think it's reasonable to say that if you want to override the markdown renderer you need to manually call it (like we do now)